roles/nvidia_gpu_exporter/tasks/main.yml

```diff
@@ -0,0 +1,52 @@
+---
+- name: Preflight
+  ansible.builtin.include_tasks:
+    file: preflight.yml
+  tags:
+    - nvidia_gpu_exporter_install
```

Comment:
```suggestion
    - nvidia_gpu_exporter
    - install
    - configure
    - run
    - nvidia_gpu_exporter_install
```

Reply:
Thanks for your detailed review.

Is there any reason that these tags are required for this new role but not `node_exporter`?

The tags are referred to `node_exporter`'s, like below, https://github.com/prometheus-community/ansible/blob/e2d8b745dc188a389300e903ee1c6e78e6cd7358/roles/node_exporter/tasks/main.yml#L5-L8

Reply:
They should be there as well, thanks for letting me know.